server/README.md

```diff
@@ -0,0 +1,16 @@
+To run backend server execute a command:
+
+1. Install all packages (only once)
+
+  ```
+  julia
+  using Pkg
+  Pkg.add("Genie")
+  Pkg.add("HTTP")
+  ```
```

Comment:
nie tak sie to robi, można tak ale trzeba uzywać tego czegos na ksztal pythonowego venva